modules/nf-core/glimpse2/concordance/main.nf

```diff
@@ -0,0 +1,55 @@
+process GLIMPSE2_CONCORDANCE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::glimpse-bio=2.0.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/glimpse-bio:2.0.0--hf340a29_0':
+        'quay.io/biocontainers/glimpse-bio:2.0.0--hf340a29_0' }"
+
+    input:
+    tuple val(meta), path(estimate), path(estimate_index), path(truth), path(truth_index), path(freq), path(freq_index), path(samples), val(region)
+    tuple val(meta2), path(groups), val(bins), val(ac_bins), val(allele_counts)
```

Comment:
This is bit of a bummer situation but I don't see a better solution indeed, good job!